clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -376,36 +376,36 @@ protected void processConsumerAction(ConsumerAction message, Store store) throws
            * so quickly skip this state transition and go straight to the final transition.
            */
           LOGGER.info(
-              "State transition from STANDBY to LEADER is skipped for topic {} partition {}, because Helix has assigned another role to this replica.",
-              topicName,
-              partition);
+              "State transition from STANDBY to LEADER is skipped for replica: {}, because Helix has assigned another role to it.",
+              Utils.getReplicaId(topicName, partition));
           return;
         }
 
         PartitionConsumptionState partitionConsumptionState = partitionConsumptionStateMap.get(partition);
         if (partitionConsumptionState == null) {
           LOGGER.info(
-              "State transition from STANDBY to LEADER is skipped for topic {} partition {}, because partition consumption state is null and the partition may have been unsubscribed.",
-              topicName,
-              partition);
+              "State transition from STANDBY to LEADER is skipped for replica: {}, because PCS is null and the partition may have been unsubscribed.",
+              Utils.getReplicaId(topicName, partition));
           return;
         }
 
         if (partitionConsumptionState.getLeaderFollowerState().equals(LEADER)) {
           LOGGER.info(
-              "State transition from STANDBY to LEADER is skipped for topic {} partition {}, because this replica is the leader already.",
-              topicName,
-              partition);
+              "State transition from STANDBY to LEADER is skipped for replica: {} as it is already the partition leader.",
+              Utils.getReplicaId(topicName, partition));
           return;
         }
         if (store.isMigrationDuplicateStore()) {
           partitionConsumptionState.setLeaderFollowerState(PAUSE_TRANSITION_FROM_STANDBY_TO_LEADER);
-          LOGGER.info("{} for partition {} is paused transition from STANDBY to LEADER", ingestionTaskName, partition);
```

Comment:
Can we get rid of `ingestionTaskName` since you have eliminited lots of usage of it?

Reply:
We still need to keep it for common resources.  Maybe in phase-II we can rewrite these logs and rely on just store name with version?

Reply:
Sure. We can do that. We have several places that use store names directly or indirectly. It'd be good to clean them up if possible!

Reply:
Yeah, I'll remove it in a follow up PR.